src/lib/_loading.scss

```diff
@@ -39,6 +39,22 @@
     }
 }
 
+.bp-thumbnails-open.bp-preload-complete {
+    pointer-events: all;
+}
+
+.bp-presentation-preload-wrapper {
+    pointer-events: all;
+}
+
+.bp-preloader-loaded {
+    margin-right: auto;
+    margin-left: auto;
+    padding: 0 20px;
```

Comment:
Where does `20px` come from? Is there a variable we can use?

Reply:
That's for the padding for the doc first images. I can create a variable for this